docs/src/test-api/class-testconfig.md

```diff
@@ -439,6 +439,14 @@ export default defineConfig({
 
 Test files that took more than `threshold` milliseconds are considered slow, and the slowest ones are reported, no more than `max` number of them. Passing zero as `max` reports all test files that exceed the threshold.
 
+## property: TestConfig.respectGitIgnore
+* since: v1.45
+- type: ?<[boolean]>
+
+Whether to skip entries from `.gitignore` when searching for test files. By default it is true, i.e. if
```

Comment:
Let's not say that by default it is true? Instead, explain that default behavior is respecting `.gitignore` when neither `TestConfig.testDir` nor `TestProject.testDir` are specified. Because that's not equal to setting the property to `true` or `false`.